Support emoji in path name for hawkeye-native image

I don't dig into details now, but it seems if the path name contain emoji, HawkEye cannot work as expected.

```
➜  ghreports git:(main) ✗ docker run --pull=always -it --rm -v $(pwd):/github/workspace -u $(id -u):$(id -g) ghcr.io/korandoru/hawkeye-native:edge check

edge: Pulling from korandoru/hawkeye-native
990ec64f338f: Pull complete 
50a474949013: Pull complete 
75b8c63a2442: Pull complete 
Digest: sha256:826ecfac070e4f5ef42b5779b90156d2f18b63beb844a01557567aa090db5e09
Status: Downloaded newer image for ghcr.io/korandoru/hawkeye-native:edge
java.lang.IllegalArgumentException: Unable to read file ./dashboard/???? Home.py
	at io.korandoru.hawkeye.core.FileContent.<init>(FileContent.java:43)
	at io.korandoru.hawkeye.core.document.Document.<init>(Document.java:48)
	at io.korandoru.hawkeye.core.document.DocumentFactory.getWrapper(DocumentFactory.java:68)
	at io.korandoru.hawkeye.core.document.DocumentFactory.createDocuments(DocumentFactory.java:51)
	at io.korandoru.hawkeye.core.LicenseProcessor.call(LicenseProcessor.java:112)
	at io.korandoru.hawkeye.command.HawkEyeCommandCheck.call(HawkEyeCommandCheck.java:45)
	at io.korandoru.hawkeye.command.HawkEyeCommandCheck.call(HawkEyeCommandCheck.java:29)
	at picocli.CommandLine.executeUserObject(CommandLine.java:1953)
	at picocli.CommandLine.access$1300(CommandLine.java:145)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2358)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2352)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2314)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2179)
	at picocli.CommandLine$RunLast.execute(CommandLine.java:2316)
	at picocli.CommandLine.execute(CommandLine.java:2078)
	at io.korandoru.hawkeye.command.HawkEyeCommandMain.main(HawkEyeCommandMain.java:35)
Caused by: java.io.FileNotFoundException: /github/workspace/./dashboard/???? Home.py (No such file or directory)
	at java.base@17.0.5/java.io.FileInputStream.open0(FileInputStream.java)
	at java.base@17.0.5/java.io.FileInputStream.open(FileInputStream.java:216)
	at java.base@17.0.5/java.io.FileInputStream.<init>(FileInputStream.java:157)
	at java.base@17.0.5/java.io.FileInputStream.<init>(FileInputStream.java:111)
	at java.base@17.0.5/sun.net.www.protocol.file.FileURLConnection.connect(FileURLConnection.java:86)
	at java.base@17.0.5/sun.net.www.protocol.file.FileURLConnection.getInputStream(FileURLConnection.java:189)
	at java.base@17.0.5/java.net.URL.openStream(URL.java:1161)
	at org.apache.commons.io.IOUtils.toString(IOUtils.java:2953)
	at org.apache.commons.io.IOUtils.toString(IOUtils.java:2910)
	at io.korandoru.hawkeye.core.FileContent.<init>(FileContent.java:41)
	... 15 more
➜  ghreports git:(main) ✗ ~/Labs/hawkeye-1.5.4/hawkeye check
[main] WARN io.korandoru.hawkeye.command.HawkEyeCommandCheck - Processing unknown files: [/Users/chenzili/Brittani/ghreports/ghreports/__pycache__/__init__.cpython-311.pyc, /Users/chenzili/Brittani/ghreports/ghreports/__pycache__/client.cpython-311.pyc, /Users/chenzili/Brittani/ghreports/ghreports/__pycache__/datamodel.cpython-311.pyc, /Users/chenzili/Brittani/ghreports/poetry.lock, /Users/chenzili/Brittani/ghreports/.env]
[main] INFO io.korandoru.hawkeye.command.HawkEyeCommandCheck - Found missing header files: [/Users/chenzili/Brittani/ghreports/dashboard/🏡 Home.py]
```

It seems like a docker mount issue.

--- a/hawkeye-core/src/test/java/io/korandoru/hawkeye/core/SelectionTest.java
+++ b/hawkeye-core/src/test/java/io/korandoru/hawkeye/core/SelectionTest.java
@@ -78,6 +78,15 @@ class SelectionTest {
         assertThat(selectedFiles).containsExactlyInAnyOrderElementsOf(expectedSelectedFiles);
     }
 
+    @Test
+    void testContainsEmoji(@TempDir Path tempDir) throws IOException {
+        final File root = createFakeProject(tempDir, new String[]{"🏡Home.py"});
+        final Selection selection = new Selection(root, new String[]{"**/*.py"}, new String[0], false);
+        final String[] selectedFiles = selection.getSelectedFiles();
+        final List<String> expectedSelectedFiles = List.of("🏡Home.py");
+        assertThat(selectedFiles).containsExactlyInAnyOrderElementsOf(expectedSelectedFiles);
+    }
+
     private File createFakeProject(Path tempDir, String[] paths) throws IOException {
         final File temp = tempDir.toFile();
         for (String path : paths) {

